3d_font.txt

```diff
@@ -363,20 +363,20 @@ z
         \ |             |
          \|_____________|
 
- 
-    
-     
-      
-       
-        
-         
-          
-           
-            
+
```

Comment:
These lines had trailing spaces which you removed :P. Could you remove this change from the commit?